### PR TITLE
[DE-248] DE-472 Unify simple error responses

### PR DESCRIPTION
**Pull request by michalpierog**

### CODE REVIEW

File: reference/Chargify-API.v1.yaml [hunk 1]
```diff
@@ -9302,17 +9206,7 @@ paths:
           content:
             application/json:
               schema:
-                description: ""
```
Comment (maciej-nedza) [inline reference/Chargify-API.v1.yaml:9305]: can we somehow preserve this example here?

File: reference/Chargify-API.v1.yaml [hunk 2]
```diff
@@ -17750,20 +17466,7 @@ paths:
...
-                    type: object
-                    properties:
-                      base:
-                        type: array
-                        items:
-                          type: string
-                x-examples:
-                  example-1:
-                    errors:
-                      base:
```
Comment (maciej-nedza) [inline reference/Chargify-API.v1.yaml:17765]: can we somehow preserve this example here?

File: reference/Chargify-API.v1.yaml [hunk 3]
```diff
@@ -17851,20 +17554,7 @@ paths:
...
```
Comment (maciej-nedza) [inline reference/Chargify-API.v1.yaml:17866]: can we somehow preserve this example here?

**Review by maciej-nedza — APPROVED** [review]
LGTM

File: reference/Chargify-API.v1.yaml [hunk 4]
```diff
@@ -16909,19 +16658,7 @@ paths:
           content:
             application/json:
               schema:
-                description: ""
-                type: object
-                x-examples:
-                  example-1:
-                    errors:
-                      - Cannot generate an invoice in advance for subscription in the group
```
[the remark this answers is not in the document]
Comment (michalpierog) [reply reference/Chargify-API.v1.yaml:16917]: There is the same example in examples section.

File: reference/Chargify-API.v1.yaml (hunk 2 above)
Comment (michalpierog) [reply reference/Chargify-API.v1.yaml:17765]: There is the same example in examples section.

File: reference/Chargify-API.v1.yaml (hunk 3 above)
Comment (michalpierog) [reply reference/Chargify-API.v1.yaml:17866]: There is the same example in examples section.

File: reference/Chargify-API.v1.yaml (hunk 1 above)
Comment (michalpierog) [reply reference/Chargify-API.v1.yaml:9305]: This was empty description.

File: spectral/schemaMustBeReused.js [hunk 5]
```diff
@@ -20,4 +28,14 @@ export default (schema, _, context) => {
       },
     ];
   }
+
+  function validateComplexSchema(complexSchema) {
+    if (complexSchema.filter((element) => !element["$ref"]).length > 0) {
+      return [
+        {
+          message: "Only $ref can be used in 'oneOf/anyOf/allOf'",
```
[the remark this answers is not in the document]
Comment (michalpierog) [reply spectral/schemaMustBeReused.js:36]: No, I will fix it later (it's now applied only to main file as it works on non resolved documents).

**Pull request merged**